sdk/communication/communication-chat/review/communication-chat.api.md

```diff
@@ -124,7 +124,7 @@ export class ChatThreadClient {
     sendTypingNotification(options?: SendTypingNotificationOptions): Promise<boolean>;
     readonly threadId: string;
     updateMessage(messageId: string, options?: UpdateMessageOptions): Promise<OperationResponse>;
-    updateThread(options?: UpdateThreadOptions): Promise<OperationResponse>;
+    updateTopic(topic: string, options?: UpdateTopicOptions): Promise<OperationResponse>;
```

Comment:
@DominikMe I am not sure should we use `request:UpdateTopicRequest` here or just `topic:string` is enough

Reply:
@LuChen-Microsoft In my opinion, if you don't expect to add more required arguments in the future, just the `topic: string` is good.

If we had >2 required arguments, I would default to use an object instead because it gives you named properties inside that object.

Reply:
good point @DominikMe . Actually we did the same on the other sdks.